BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -279,4 +283,26 @@ extension TipsDetailViewController {
       })
     }
   }
+  
+  func setupLedgerObservers() {
+    ledgerObserver.recurringTipRemoved = { [weak self] key in
+      guard let self = self, self.isViewLoaded else {
+        return
+      }
+      let tableView = self.tipsView.tableView
+      let oldCount = self.tipsList.count
+      self.tipsList.removeAll(where: {
+        $0.category == RewardsCategory.recurringTip.rawValue &&
+        $0.id == key
+      })
+      if oldCount > self.tipsList.count {
+        var sections: IndexSet = [Section.tips.rawValue]
+        if tableView.numberOfRows(inSection: Section.summary.rawValue) == 2 &&
+          !self.tipsList.contains { $0.rewardsCategory == .recurringTip } {
+          sections.insert(Section.summary.rawValue)
+        }
+        self.tipsView.tableView.reloadSections(sections, with: .automatic)
```

Comment:
still animating this, I think the whole section here can be replaced right?

Reply:
This is a different controller, should I remove animation here too?

Reply:
Yeah lets just do all of it